test/unit/test/sequence_generator_test.cpp

```diff
@@ -0,0 +1,146 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+
+#ifdef SEQAN3_HAS_SEQAN2
+#include <seqan/sequence.h>
+// #include <seqan/stream.h>
+#endif
+
+using seqan3::operator""_dna4;
+using seqan3::operator""_dna5;
+
+// generate (single) fixed sized sequence
+TEST(random_sequence, generate_random_sequences_with_fix_length)
+{
+    std::mt19937_64 random_engine1{1};
+    std::mt19937_64 random_engine2{2};
+
+    seqan3::test::random_sequence_generator<seqan3::dna4_vector> random_sequence_gen{3};
+
+    seqan3::dna4_vector sequence1 = random_sequence_gen(random_engine1); // ACA
+    seqan3::dna4_vector sequence2 = random_sequence_gen(random_engine2); // TTT
```

Comment:
```suggestion
    seqan3::dna4_vector sequence2 = random_sequence_gen(random_engine2); // TTT
    seqan3::dna4_vector sequence3 = random_sequence_gen(random_engine1); // ???
```

we should test that the generator is advanced correctly and does not always return the same value.